python/lsst/meas/extensions/shapeHSM/_hsm_moments.py

```diff
@@ -88,13 +83,13 @@ def calculate(
                 badpix=badpix,
                 guess_sig=sigma,
                 precision=1.0e-6,
```

Comment:
Make `precision` a config as well in the base class, with `1e-6` as the default value.